specification/monitor/resource-manager/readme.md

```diff
@@ -27,16 +27,26 @@ These are the global settings for the MonitorClient API.
 ``` yaml !$(python) || !$(track2)
 title: MonitorClient
 ```
+
 ``` yaml $(python) && $(track2)
 title: MonitorManagementClient
 ```
 
 ``` yaml
 description: Monitor Management Client
 openapi-type: arm
-tag: package-2020-03
+tag: package-2016-06
```

Comment:
```suggestion
tag: package-2020-03
```

Reply:
Thanks for reviewing, I've incorporated your suggestion.